app/views/api/v1/split_details/show.json.jbuilder

```diff
@@ -1 +1,2 @@
 json.(@split_detail, :name, :hypothesis, :assignment_criteria, :description, :owner, :location, :platform)
+json.variant_details @split_detail.split.variant_details, partial: 'variant_detail', as: :variant_detail
```

Comment:
you can do that. you'll also want to add `includes(:variant_details)` wherever the split is being queried